webpack/scenes/ModuleStreams/ModuleStreamsPage.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, Col, Form, FormGroup } from 'react-bootstrap';
+import qs from 'query-string';
+import Search from '../../components/Search/index';
+import ModuleStreamsTable from './ModuleStreamsTable';
+import { orgId } from '../../services/api';
+
+class ModuleStreamsPage extends Component {
+  constructor(props) {
+    super(props);
+
+    const queryParams = qs.parse(this.props.location.search);
+    this.state = {
+      searchQuery: queryParams.search || '',
+    };
+  }
+
+  componentDidMount() {
+    this.props.getModuleStreams({
+      search: this.state.searchQuery,
+    });
+  }
+
+  onPaginationChange = (pagination) => {
+    this.props.getModuleStreams({
+      ...pagination,
+    });
+  };
+
+  onSearch = (search) => {
+    this.props.getModuleStreams({ search });
+  };
+
+  getAutoCompleteParams = search => ({
+    endpoint: '/module_streams/auto_complete_search',
+    params: {
+      organization_id: orgId(),
+      search,
+    },
+  });
+
+  updateSearchQuery = (searchQuery) => {
+    this.setState({ searchQuery });
+  };
+
+  render() {
+    const { moduleStreams } = this.props;
+    return (
+      <Grid bsClass="container-fluid">
+        <Row>
+          <Col sm={12}>
+            <h1>{__('Module Streams')}</h1>
+          </Col>
+        </Row>
+        <Row>
+          <Col sm={6}>
+            <Form className="toolbar-pf-actions">
+              <FormGroup className="toolbar-pf-filter">
```

Comment:
Should also add `toolbar-pf` to the `className` (it will add some `padding-top`)

Reply:
@amirfefer bionic eyes cached it :eye:

Reply:
To test you need pulp 2.17 and sync -> https://repos.fedorapeople.org/pulp/pulp/fixtures/rpm-with-modules/